src/core/Akka.Tests/Actor/FailingActorSpec.cs

```diff
@@ -0,0 +1,490 @@
+            {
+                child = Context.ActorOf(new RandomPool(1).WithSupervisorStrategy(strategy).Props(Props.Create<FailingActor>()));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreatePooledChildConsistentHashing()
+            {
+                child = Context.ActorOf(new ConsistentHashingPool(1).WithHashMapping(o => o.GetHashCode()).WithSupervisorStrategy(strategy).Props(Props.Create<FailingActor>()));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreatePooledChildTailChopping()
+            {
+                child = Context.ActorOf(new TailChoppingPool(1, TimeSpan.FromSeconds(10), TimeSpan.FromMilliseconds(20)).WithSupervisorStrategy(strategy).Props(Props.Create<FailingActor>()));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreatePooledChildScatterGatherFirstCompleted()
+            {
+                child = Context.ActorOf(new ScatterGatherFirstCompletedPool(1).WithSupervisorStrategy(strategy).Props(Props.Create<FailingActor>()));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreatePooledChildSmallestMailbox()
+            {
+                child = Context.ActorOf(new SmallestMailboxPool(1).WithSupervisorStrategy(strategy).Props(Props.Create<FailingActor>()));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreateGroupedChildBroadcast()
+            {
+                IActorRef realChild = Context.ActorOf(Props.Create<FailingActor>(), "failing");
+
+                child = Context.ActorOf(Props.Empty.WithRouter(new BroadcastGroup(new string[] { realChild.Path.ToString() })).WithSupervisorStrategy(strategy));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreateGroupedChildRoundrobin()
+            {
+                IActorRef realChild = Context.ActorOf(Props.Create<FailingActor>(), "failing");
+
+                child = Context.ActorOf(Props.Empty.WithRouter(new BroadcastGroup(new string[] { realChild.Path.ToString() })).WithSupervisorStrategy(strategy));
+
+                Receive<string>(m => child.Tell(m), m => true);
+
+                resetEvt.Set();
+            }
+
+            private void CreateGroupedChildRandom()
+            {
+                IActorRef realChild = Context.ActorOf(Props.Create<FailingActor>(), "failing");
+
+                child = Context.ActorOf(Props.Empty.WithRouter(new BroadcastGroup(new string[] { realChild.Path.ToString() })).WithSupervisorStrategy(strategy));
```

Comment:
The `SupervisionStrategy` isn't being used on any of these group router tests at the moment, because group routers aren't parents. They're just actors that use actor selections to send messages to other actors created by someone else. You'll need to put a parent actor with the specified supervision strategy above the child. Looks like this is true for all of the specs.